tests/data/placebo/test_s3_encrypt/s3.CopyObject_1.json

```diff
@@ -3,21 +3,31 @@
     "data": {
         "CopyObjectResult": {
             "LastModified": {
-                "hour": 9, 
+                "hour": 23, 
                 "__class__": "datetime", 
-                "month": 5, 
-                "second": 15, 
+                "month": 12, 
+                "second": 29, 
                 "microsecond": 0, 
                 "year": 2016, 
-                "day": 8, 
-                "minute": 39
+                "day": 20, 
+                "minute": 43
             }, 
             "ETag": "\"c68271a63ddbc431c307beb7d2918275\""
         }, 
         "ResponseMetadata": {
             "HTTPStatusCode": 200, 
-            "HostId": "L1ZMtrOGwF+vkecrlREyjgBXICb17QQzgslMtlD2cuw/wPfW2WQdduJCdv0SihxDegQqekKx56g=", 
-            "RequestId": "DBCAF1251C0F0B79"
+            "RetryAttempts": 0, 
+            "HostId": "YrgyrXKdjgYLs09ixH8UUOpZMhK7VaqpoV/7lZZjIUr3VY2VD4SP0NW2h+/nNW9lI8sK5dYzUX8=", 
+            "RequestId": "D4AABBE372B7F5EB", 
+            "HTTPHeaders": {
```

Comment:
nah i wouldn't worry about the headers, occasionally its nice to strip out extraneous buckets in the reply, but the biggest gain is just avoiding unesc api calls when possible vs stripping individual call outputs. again its fine for this pr, just want awareness for future.